### ISSUE TITLE
[Bug]: Skjult element har fokuserbart innhold (kart på forsiden)

### ISSUE THREAD

**Issue by KjerstiSteien** [Informasjonsforvaltning/fdk-frontend#313]
## 🐛 Bug rapport
Oppdaget gjennom Siteimprove

### Nåværende oppførsel
Skjult element har fokuserbart innhold. 

Et sideelement med attributtet aria-hidden="true" inneholder fokuserbare elementer (elementer som kan motta fokus fra et tastatur)

Kodelinjene Siteimprove reagerer på:

<div class="norgeskart_norgeskart__X1KV1 leaflet-container leaflet-touch leaflet-fade-anim leaflet-grab leaflet-touch-drag leaflet-touch-zoom" aria-hidden="true" tabindex="0">
<div class="leaflet-pane leaflet-map-pane" style="transform: translate3d(0px, 0px, 0px);">  </div>
<div class="leaflet-control-container">  </div>
</div>

Når jeg tester ved å tabbe gjennom forsiden ser jeg at det er et mellomsteg jeg ikke kan se mellom "Om KI-søket" og pilen som fører meg ned til mer informasjon på forsiden. 
### Forventet oppførsel
Skjulte elementer skal ikke ha fokuserbart innhold. 

### Hvordan reprodusere?
Tab gjennom forsiden. Se at du hopper innom et skjult punkt mellom "Om KI-søket" og pilen som fører deg ned til mer informasjon på forsiden. 

Utforsk problemet gjennom Siteimprove:
https://my2.siteimprove.com/Inspector/1146729/A11Y/Page?pageId=9254141986&impmd=4B718E82813DDA6397FABFA3A1B3D934&conf=a+aa&issue=cantTell+failed&wcag=twopointone#/sia-r17/failed/elVaaDUydEg=;ZDE5MTUyMGE=
